docs/state.rst

```diff
@@ -91,8 +91,11 @@ and the third argument is the value to write. For example:
 .. code-block:: python
 
     App.localPut(Int(0), Bytes("role"), Bytes("admin")) # write a byte slice to Txn.accounts[0], the sender's account
+    App.localPut(Txn.sender(), Bytes("role"), Bytes("admin")) # write a byte slice to the sender's account
```

Comment:
Same thing as the above comment, I think all examples should only use direct references. In this file you will need to modify some of the text in addition to the examples too.

Reply:
Document updated.